src/documentation/WizardForm/Wizard.js

```diff
@@ -1,89 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Form, FormSpy } from 'react-final-form';
 import FormWizard from '../../components/FormWizard';
 import FormControls from '../../components/FormControls';
 import StepNavigation from '../../components/StepNavigation';
 import StepNavigationItem from '../../components/StepNavigationItem';
 
-const handleValidation = props => {
+const handleValidation = (props) => {
   return null;
 };
 
-export default class Wizard extends React.Component {
-  static propTypes = {
-    onSubmit: PropTypes.func.isRequired,
-  };
-  static Page = ({ children }) => children;
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      page: 0,
-      values: props.initialValues || {},
-    };
+const Page = ({children}) =>{
+  return children;
+}
+
+
+const Wizard =({children, initialValues, onSubmit, ...other })=>{
+  
+  
+  const [page, setpage] = useState(0);
+  const [values, setvalues] = useState(initialValues || {})
+
+  const handleTabClick = (values) =>{
+    setpage(values);
+    setvalues(values);
   }
 
-  handleTabClick = values =>
-    this.setState(state => ({
-      page: values,
-      values,
-    }));
+  
+  const next = (values) =>{
+    setpage(Math.min(page + 1, children.length - 1));
```

Comment:
Please use camelcase for the naming off the set functions, e.g. `setPage` Thank you!

Reply:
sure